src/common/models/Projection.ts

```diff
@@ -199,9 +199,7 @@ export class Projection {
       summaryWithTimeseries.actuals.cumulativeDeaths;
     this.currentCumulativeCases =
       summaryWithTimeseries.actuals.cumulativeConfirmedCases;
-    this.currentContactTracerMetric = this.contractTracers
-      .filter(x => x !== null)
-      .slice(-1)[0];
```

Comment:
This ended up setting currentContactTracingMetric to `undefined` instead of `null` so I changed it to use the existing `lastValue` helper.

Reply:
I can't remember why i used this initially I think there was some like undefined vs null situation